template.yaml

```diff
@@ -21,6 +21,11 @@ Resources:
       WebsiteConfiguration:
         IndexDocument: index.html
         ErrorDocument: error.html
+        RoutingRules:
+          - RedirectRule:
```

Comment:
Out of curiousity: was this needed for the case when the user writes something random as sessionId?

Reply:
No, it should redirect 404 ot the index html